### DISCUSSION TITLE
Using a custom head model in simNIBS

### DISCUSSION THREAD

**Discussion by AlihanKayabas** [simnibs/simnibs#480] [7.Creating head models]
Hello all,

After running headreco (or charm) in SimNIBS, the output includes a head model and an m2m folder, which I can load into the SimNIBS GUI to run tDCS simulations and generate E-field outputs.  However, for my pipeline, I need to use a custom head model that I have already computed in Brainstorm.  

Here’s my specific issue:  
- I have reduced the number of vertices in the Brainstorm head model to 15,000 (down from the ~160,000 vertices ). This reduction is necessary for my pipeline.
- I have not been able to find a method to import or integrate this reduced head model into SimNIBS.

Is there a way to achieve this, or could you provide guidance on how to modify the SimNIBS workflow to accommodate externally reduced head models? Any suggestions would be greatly appreciated.  

What I would like to achieve is running tDCS simulations on a head model with reduced number of vertices.

**Simply put, is there a "less vertices" option available in SimNIBS, equivalent to the one in Brainstorm?**

Thank you very much for your time and assistance.  

Kind regards
